projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -100,6 +108,7 @@ export class CommentComponent implements OnChanges {
   public onSave() {
     this._comment.content = this.fullComment.substring(0, this.COMMENT_CHAR_LIMIT);
     this.updated.emit(this._comment);
+    this.unsavedChanges.emit(false);
```

Comment:
i think this might not be the best place for this. including the one in on cancel, my thinking is, even if you click save, you dont know if the api called passed or failed so the comment will be there and this would emit incorrectly.